db/articles.js

```diff
@@ -0,0 +1,76 @@
+let articles = [];
+
+function all() {
+  return articles;
+}
+
+function createURLTitle(title) {
+  let newTitle = encodeURIComponent(title);
+  return newTitle;
+}
+
+function checkArticleExists(inputTitle){
+  for(let i=0; i<articles.length; i++){
+    let current = articles[i];
+    if(current.title === inputTitle){
+      return true
+    }
+  }
+  return false;
+}
+
+function findArticleByTitle(inputTitle) {
+  let article = {};
+  for(let i=0; i<articles.length; i++){
+    let current = articles[i];
+    if(current.title === inputTitle){
+      article = current
+    }
+  }
+
+  if(article === {}){
+    return false;
+  }
+
+  return article;
+}
+
+function addArticle(responseObject) {
+  let newArticle = {};
+  newArticle.urlTitle = createURLTitle(responseObject.title);
+  newArticle.title = responseObject.title;
+  newArticle.body = responseObject.body;
+  newArticle.author = responseObject.author;
+
+  articles.push(newArticle);
+  return;
```

Comment:
Doesn't hurt to have a `return` here and be explicit, but this is essentially doing `return undefined`. That is also the implicit behavior if *not* using a `return` statement at all.